Makefile

```diff
@@ -1,4 +1,18 @@
 
+.PHONY: default
+default:
+	@echo "Desk doesn't need to be compiled. Please run \"sudo make install\" to install desk." ||:
```

Comment:
Great, thanks for the explanation. Yeah, I'd say let's strip the `||:` off the echo lines, then we'll be good to go.